src/video_analyze.py

```diff
@@ -766,7 +766,7 @@ def get_options(argv):
     parser.add_argument(
         "--quiet",
         action="store_const",
-        dest="debug",
+        dest="quiet",
```

Comment:
This is wrong, right? "--quiet" is a shortcut for "no debug info at all". That's why we're setting the debug level variable to -1